docs/rules.md

```diff
@@ -0,0 +1,109 @@
+# oVirt to Kubevirt rules to import virtual machine
+
+## Actions
+
+Each validation failure will result in a predefined action being performed.
+
+The description of the actions is following:
+* **Log** - rule validation failure will result in logging that information to operator logs and the CR will be processed;
+* **Warn** - rule validation failure will result in saving that information in the CR and the CR will be processed (import will be executed);
+* **Block** - rule validation failure will result in savint that information in the CR and the CR will not be processed (import will not be executed).
+
+## Networking rules
+
+### Preconditions
+
+1. Network to be used by the imported virtual machine has been properly pre-configured in the target cluster by its administrator (host network configuration and network attachment definition).
+2. SR-IOV network interfaces are not supported at this stage.
```

Comment:
We need to remember to remove that after #112 is implemented; I've added comment to the issue.

Reply:
:+1: